src/bin/ayb.rs

```diff
@@ -214,19 +236,78 @@ async fn main() -> std::io::Result<()> {
                         }
                     }
                 }
+            } else if let Some(matches) = matches.subcommand_matches("profile") {
+                if let (Some(entity), Some(format)) = (
+                    matches.get_one::<String>("entity"),
+                    matches.get_one::<OutputFormat>("format"),
+                ) {
+                    match client.entity_details(entity).await {
+                        Ok(response) => match format {
+                            OutputFormat::Table => response.profile.generate_table()?,
+                            OutputFormat::Csv => response.profile.generate_csv()?,
+                        },
+                        Err(err) => println!("Error: {}", err),
+                    }
+                }
+            } else if let Some(matches) = matches.subcommand_matches("update_profile") {
+                if let Some(entity) = matches.get_one::<String>("entity") {
+                    let update_profile = {
+                        match client.entity_details(entity).await {
+                            Ok(response) => {
+                                let current_profile = response.profile;
+                                let update_profile = ProfileUpdate {
```

Comment:
Given it's a `PATCH`, should we make it so the endpoint accepts a subset of the arguments and handles filling in the previous values on the server side? So I can send only a description or organization or links, and that's OK.

Reply:
You're right, that was left from the `POST` implementation :p
I'll update it :)